### ISSUE TITLE
Noticeable chess board effect on Sintel pre-trained checkpoints output

### ISSUE THREAD

**Issue by chenusc11** [daigo0927/tf-raft#23]
Hi, @daigo0927 thank you for your decent implementation of the TF version. It was fun playing with it.

However, I noticed that with your pre-trained Sintel checkpoints. It tends to produce Chess Board Effect. (TF version (top) vs. Paper Pytorch version (bottom))
![frame_0016_0017_tf-raft](https://user-images.githubusercontent.com/8814381/99815853-cb163c00-2b42-11eb-9003-a84c497b06c4.png)
![frame_0017_vsGT_opticalflow](https://user-images.githubusercontent.com/8814381/99815879-d4070d80-2b42-11eb-8da1-dbf1a21abbae.png)

Do you know where does it come from? Thanks!

**Comment by daigo0927** [comment]
Sorry for my late reply...

This is caused by the difference in the training procedure between mine and the original one. The training reported in the original paper uses FlyingChairs and FlyingThigs3D datasets (see Table1 in https://arxiv.org/abs/2003.12039) for pre-training.
I have trained the RAFT model only on MPI-Sintel or FlyginChairs, and FlyingThings3D is too large for me to train the model in my personal GCP environment 💸 ... This lack of training may cause the above checker-board effect.

I have described other notes at the bottom of the README. Thanks! 😄
